### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by victor-ciurel**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -91,14 +92,14 @@
...
+        "jest": "^24.0.0",
+        "lint-staged": "^8.1.1",
         "prettier": "^1.9.2",
-        "prettier-standard": "^8.0.1",
+        "prettier-standard": "^9.1.1",
         "prop-types": "^15.6.0",
-        "react": "16.2.0",
-        "react-dom": "16.2.0",
+        "react": "16.7.0",
+        "react-dom": "16.7.0",
```
Comment (mihaimaruseac) [inline package.json]: Thanks for fixing [this](https://github.com/rosedu/site-ng/network/alert/package.json/react-dom/open)

Comment (victor-ciurel) [reply package.json]: Well actually. Everything breaks now due to new gatsby version. Will try to fix. Or just upgrade selectively.

Comment (mihaimaruseac) [reply package.json]: Haskell for the win :P

Comment (victor-ciurel) [reply package.json]: Yeah... how about no.

**Comment by mihaimaruseac** [comment]
Closing as stale. Too many merge conflicts.

**Pull request closed without merging**